src/algebra/algebra/spectrum.lean

```diff
@@ -245,6 +243,84 @@ begin
     exact ⟨unit_mem_mul_iff_mem_swap_mul.mp k_mem, k_neq⟩ },
 end
 
+open polynomial
+/-- This is half of the spectral mapping theorem for polynomials. We prove it separately
+because it holds over any field, whereas `spectrum.polynomial_spectrum` needs the field to
+be algebraically closed. -/
+theorem polynomial_subset (a : A) (p : polynomial 𝕜) :
```

Comment:
I think a more accurate name would be:
```suggestion
theorem subset_polynomial_aeval (a : A) (p : polynomial 𝕜) :
```

Reply:
my reasoning (here and elsewhere in this file) for _not_ including the `aeval` was that declarations should sort of involve the minimal amount of context to make sense of the thing: hence `spectrum.polynomial_subset`. Am I wrong about this reasoning (according to mathlib style), or is this just a matter of taste.